src/Components/Mission/Mission.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import MissionList from './missionList';
+import { getMissions, getJoined, switchMissions } from '../Redux/Mission/Actions/missionAction';
+
+const Mission = () => {
+  const dispatch = useDispatch();
+  const missions = useSelector((state) => state.missions);
+  useEffect(() => {
+    if (missions.length === 0) dispatch(getMissions());
+    else dispatch(getJoined(missions));
```

Comment:
"Join Mission" button updates the store successfully by dispatching an action 💪😎 Well done ✔

Reply:
@mateo951 Thank you for the approval